### ISSUE TITLE
[Pottential Conflict and Game Crash] Ender I/O Fluid Conduit/Immersive Engineering Fluid Pipes

### ISSUE THREAD

**Issue by JTKillien** [scalda/Foolcraft_3#181]
## Modpack Information
- [x] I am running the latest version of FoolCraft
- [x] I am running FoolCraft via Curse/Twitch Launcher
- [x] I can reproduce this issue consistently
     - [x] In single player
     - [ ] In multiplayer
- [x] I have searched for this issue previously and it was either (1) not previously reported, or (2) previously fixed and I'm having the same problem.
- [x] I am crashing and can provide my crash report(s)
- [ ] I have not altered the modpack [if you have, note the removed mods/added mods/changes in Additional Information]

## Issue Description 
In the newest update (1.4.4) the game immediately crashes upon placing Immersive Engineering Fluid Pipes next to Ender IO Fluid conduits. I have tested this multiple times via creative world and my survival world. In survival the world crashes upon loading without remove the Fluid pipes via MC Edit, and creative confirmed that placing the Fluid pipes next to Ender IO Fluid conduits (weather connected or not) causes the crash.

## Reproduce Steps
Start a world, place Immersive Engineering Fluid pipe then place an Ender IO fluid conduit next to it. Game will crash.

## Additional Information
Pack Changes: Added OptiFine_1.12.2_HD_U_E1 and JEHC, disabled Better Foliage 

Crash Reports form Survival World
Upon load from update: https://pastebin.com/QGepgEZp
Upon placing fluid pipe after removal via MC Edit: https://pastebin.com/ZvNP7JHk

Crash Report form Creative Test World
Upon Placing EIO Fluid Conduit: https://pastebin.com/smck20r8

**Comment by scalda** [comment]
Remove optifine

**Comment by JTKillien** [comment]
Removed Optifine, started new world in creative and re-attempted. Crash still occurs, same issue shown. 
Crash Log: https://pastebin.com/R0aJCRh3

**Comment by Arcarda** [comment]
I also had this exact issue. What I did was use a /fill command on my server and used the co-ords that were given and then played Server Admin battleship. Anyways, it seems to be a mix of steam and pipes. Specifically immersiveengineering:metal_device1 6 with steam. This was an issue in both 1.4.4 AND 1.4.5. I can provide all client side crash reports if requested. I attempted with no additional mods as well as with mods added. All results the same until those blocks were removed. We are extensively testing now but no one has crashed since.
